### PR TITLE
Remove insensitive wordings from the docs like just - easy or - simple

### PR DESCRIPTION
**Pull request by m14815**
#### What type of PR is this?
/kind documentation

#### What this PR does / why we need it:
Clean up some insensitive wordings from docs to improve readability
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by BenTheElder** [comment]
forwarded to wg/naming.

aside:
> Fixes #17070

#17070 is a seemingly unrelated pull-request, I suspect this is not what you intended to write.
GitHub will try to close issues linked with "fixes #..." so, if so, please correct this, thanks!

File: cmd/kube-scheduler/app/server.go [hunk 1]
```diff
@@ -111,7 +111,9 @@ for more information about scheduling and the kube-scheduler component.`,
 		fmt.Fprintf(cmd.OutOrStdout(), "%s\n\n"+usageFmt, cmd.Long, cmd.UseLine())
 		cliflag.PrintSections(cmd.OutOrStdout(), namedFlagSets, cols)
 	})
-	cmd.MarkFlagFilename("config", "yaml", "yml", "json")
```
Comment (BenTheElder) [inline cmd/kube-scheduler/app/server.go:114]: this change does not seem related to the rest of the PR

File: cmd/kubeadm/app/cmd/phases/upgrade/node/kubeletconfig.go [hunk 2]
```diff
@@ -73,7 +73,6 @@ func runKubeletConfigPhase() func(c workflow.RunData) error {
 		}
 
 		// TODO: Checkpoint the current configuration first so that if something goes wrong it can be recovered
-
```
Comment (BenTheElder) [inline cmd/kubeadm/app/cmd/phases/upgrade/node/kubeletconfig.go:76]: this change seems unrelated to the rest of the PR.

**Comment by m14815** [comment]
> forwarded to wg/naming.
> 
> aside:
> 
> > Fixes #17070
> 
> #17070 is a seemingly unrelated pull-request, I suspect this is not what you intended to write.
> GitHub will try to close issues linked with "fixes #..." so, if so, please correct this, thanks!

Oh, sorry, I think I misused it, removed the number.

File: cluster/log-dump/log-dump.sh [hunk 3]
```diff
@@ -497,7 +497,7 @@ function dump_nodes() {
 # does not run on Windows nodes.
 #
 # Note: This step is O(#nodes^2) as we check if each node is present in the list of succeeded nodes.
-# Making it linear would add code complexity without much benefit (as it just takes ~1s for 5k nodes).
+# Making it linear would add code complexity without much benefit (as it takes ~1s for 5k nodes).
```
Comment (BenTheElder) [inline cluster/log-dump/log-dump.sh:500]: This is talking about how long an automated process takes, not say, how difficult something is. 
The fact that it only takes ~1s for 5k nodes seems actually relevant.

I don't think any of the lines changed in this PR are discussing e.g. difficulty, "just" is being used as a synonym for "only" when discussing the limits of something (lack of additional functionality, limited time consumed computing something, etc.)

Comment (m14815) [reply cluster/log-dump/log-dump.sh:500]: I saw an issue in kubernetes/website, is about removing this kind of words, the issue number if 17070. I may misunderstand it.

Comment (BenTheElder) [reply cluster/log-dump/log-dump.sh:500]: To be extra clear: 
- Thanks for doing this! Appreciate the help cleaning this sort of thing up ❤️ 
- I'm not sure this is _quite_ the set of changes we want, but this is only my suggestion and I'm going to ultimately defer to the wg/naming leadership (pinged in #wg-naming) 😅, I think some of the other unrelated changes should clearly be dropped though / moved to a different PR (https://github.com/kubernetes/kubernetes/pull/99548#discussion_r584255975)

**Comment by jpbetz** [comment]
/approve
For etcd image changes

**Comment by fedebongio** [comment]
Thank you Ben!

File: cluster/log-dump/log-dump.sh (hunk 3 above)
Comment (celestehorgan) [reply cluster/log-dump/log-dump.sh:500]: @BenTheElder Not to be a stick in the mud: I'd err on the side of keeping this change in :)

File: cmd/kube-scheduler/app/server.go (hunk 1 above)
Comment (celestehorgan) [reply cmd/kube-scheduler/app/server.go:114]: +1. 

@m14815 please remove this change from the PR. The commit message will be inaccurate if there are code as well as language changes.

**Review by celestehorgan — CHANGES_REQUESTED** [review]
Removals of the word "just" are LGTM, but please remove the one code change from this PR.

**Comment by m14815** [comment]
I delete the original fork, any method to recover it, then I can rebase.

**Comment by justaugustus** [comment]
@kubernetes/sig-docs-en-owners -- Are one of you able to assist w/ the git magic to recover this fork?

File: cluster/log-dump/log-dump.sh (hunk 3 above)
Comment (BenTheElder) [reply cluster/log-dump/log-dump.sh:500]: [resolving this in favor]

**Comment by BenTheElder** [comment]
If you have the changes locally still you can just fork on github again to the same location and re-push the branch to your fork.

Otherwise since the diff is still available to view in this PR and is small (in terms of number of lines changed), I would suggest the most straightforward approach is to create a new fork and just manually re-apply these changes.

https://docs.github.com/en/github/administering-a-repository/restoring-a-deleted-repository covers when a fork can be recovered through github.

Alternatively you can try to apply https://github.com/kubernetes/kubernetes/pull/99548.patch to a new local branch, but this branch already needed a rebase for the unrelated code change(s).

**Comment by m14815** [comment]
Methods are not suit to me, may I close this PR and recreate a new one?

**Comment by BenTheElder** [comment]
Sure 👍

**Comment by m14815** [comment]
> Sure 👍

Ok, Thanks. I am going to do re-PR.

**Comment by BenTheElder** [comment]
/close
superceded by https://github.com/kubernetes/kubernetes/pull/102080
